springfield/settings/__init__.py

```diff
@@ -22,17 +22,64 @@
 # CSP settings, expanded upon later:
 # NOTE: We are providing all settings to django-csp as sets, not lists.
 # - This is for de-duping, and because django-csp will convert them to `sorted` lists for us.
+
+# NOTE: Any URLs that contain a path, not just the origin, trailing slashes are important.
+# - if no path is provided, all resources are allowed from the origin.
+# - if path is provided with no trailing slash: an exact-match is required.
+#   - e.g. `https://example.com/api` will only match `https://example.com/api`
+# - if path is provided with trailing slash: the path is a prefix-match.
+#   - e.g. `https://example.com/api/` will match anything that starts with `https://example.com/api/`
+
 _csp_default_src = {
+    # Keep `default-src` minimal. Best to set resources in the specific directives.
     csp.constants.SELF,
-    "*.firefox.com",
```

Comment:
Just to double-check: we're able to drop this because it's effectively going to be the same as `SELF` yeah?

Reply:
yes, but this also removes the wildcard which is more safe. If we did need something like `api.firefox.com` it's better to add that explicitly. Also, with these changes most of the main fetch directives have values so won't use `default-src` as a fallback. Safest is to set it to `'none'` and adjust the specific directives but I didn't want to jump all the way there in this round of clean-up.